Portal/src/Datahub.Application/Services/Cost/IWorkspaceCostManagementService.cs

```diff
@@ -102,10 +102,15 @@ public class DailyServiceCost
         public string ResourceGroupName { get; set; } = String.Empty;
```

Comment:
DailyServiceCost should prob move outside IWorkspaceCostManagementService - I don't mind having multiple classes in a single file, but maybe we can move these small helpers into SupportingObjects.cs or so